File: corpus/code/email.php
<?php

function looks_like_email(string $value): bool
{
    $cleaned = trim($value);
    return str_contains($cleaned, '@') && str_contains($cleaned, '.');
}

function sum_values(array $values): int
{
    $total = 0;
    foreach ($values as $value) {
        $total += $value;
    }
    return $total;
}

File: corpus/code/email.go
package corpus

import "strings"

func LooksLikeEmail(value string) bool {
	cleaned := strings.TrimSpace(value)
	return strings.Contains(cleaned, "@") && strings.Contains(cleaned, ".")
}

func SumValues(values []int) int {
	total := 0
	for _, value := range values {
		total += value
	}
	return total
}
